third_party/amd/lib/TritonAMDGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -191,14 +191,16 @@ template <typename FPType> struct FPTypeInfo {
 template <typename ConvertOp>
 static SmallVector<Value>
 cvtScalePkUpcastFromFp8(Location loc, ConversionPatternRewriter &rewriter,
-                        Value v0, Value v1) {
+                        const SmallVector<Value> v) {
```

Comment:
We can use `ArrayRef` here--using `const SmallVector<Value> v` would make it copy the whole vector under the hood for the parameter.